src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -79,7 +79,7 @@ class TransferBalancePage extends React.Component {
                 title: this.props.translate('transferAmountPage.instant'),
                 description: this.props.translate('transferAmountPage.instantSummary', {
                     amount: this.props.numberFormat(
-                        0.25,
+                        CONST.WALLET.TRANSFER_METHOD_TYPE_FEE.INSTANT.MINIMUM_FEE / 100,
```

Comment:
This needs to be changed to say `1.5% fee ($0.25 minimum)`

Reply:
(and translated)

Reply:
Bumping this and then we should be good!

Reply:
This is being translated correctly as you asked. Its taken rate 1.5% and minimum fee $0.25 from constants.
```
this.props.translate('transferAmountPage.instantSummary', {
                    rate: this.props.numberFormat(CONST.WALLET.TRANSFER_METHOD_TYPE_FEE.INSTANT.RATE),
                    minAmount: this.props.numberFormat(
                        CONST.WALLET.TRANSFER_METHOD_TYPE_FEE.INSTANT.MINIMUM_FEE / 100,
                        {style: 'currency', currency: 'USD'},
                    ),
                }),
 ```     
 ```instantSummary: ({rate, minAmount}) => `${rate}% fee (${minAmount} minimum)```
 
 @stitesExpensify